### ISSUE TITLE
wasm-build seems to not work with the latest nightly cargo

### ISSUE THREAD

**Issue by pepyakin** [paritytech/wasm-utils#33]
This code succeeds:

```shell
rustup override set nightly-2017-10-08
cargo clean
./build.sh
```
When the following code fails

```shell
rustup override set nightly
cargo clean
./build.sh
```

 with a message:
```
thread 'main' panicked at 'Failed to process cargo target directory: NoSuitableFile("./target/wasm/wasm32-unknown-emscripten/release/deps/wasm_demo-*.wasm")', src/libcore/result.rs:860:4
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

**Comment by pepyakin** [comment]
target directory layout for nightly-2017-10-08

```
$ find target 
...
target/wasm/wasm32-unknown-emscripten/release/deps/wasm_demo-7c0e598c3a169346.js
target/wasm/wasm32-unknown-emscripten/release/deps/wasm_demo-7c0e598c3a169346.wasm
...
target/wasm/wasm32-unknown-emscripten/release/wasm-demo.js
```

And this is for the latest nightly:

```
...
target/wasm/wasm32-unknown-emscripten/release/deps/wasm-demo.js
target/wasm/wasm32-unknown-emscripten/release/deps/wasm_demo.wasm
...
target/wasm/wasm32-unknown-emscripten/release/wasm-demo.js
target/wasm/wasm32-unknown-emscripten/release/wasm_demo.wasm
```

**Comment by pepyakin** [comment]
Closed by #36
